fe/fe-core/src/main/java/com/starrocks/analysis/CreateFunctionStmt.java

```diff
@@ -58,18 +67,38 @@
     public static final String FINALIZE_KEY = "finalize_fn";
     public static final String GET_VALUE_KEY = "get_value_fn";
     public static final String REMOVE_KEY = "remove_fn";
+    public static final String OBJECT_TYPE_KEY = "object_type";
```

Comment:
If you modify stynax to `type`, there always need change?

Reply:
Yes, maybe you are reviewing old code. I have updated it.